README.md

```diff
@@ -1,36 +1,55 @@
-# eZ Platform Developer Documentation
+# Ibexa DXP Developer Documentation
 
-This repository is the source for the [developer documentation for Ibexa DXP](https://doc.ibexa.co/en/latest),
+This repository is the source for the [developer documentation for Ibexa](https://doc.ibexa.co/en/latest),
 a DXP platform that is based on the Symfony Full Stack Framework in PHP.
```

Comment:
```suggestion
a digital experience platform that is based on the Symfony Full Stack Framework in PHP.
```

Since we have DXP just before this instance, and P in DXP stands for platform, we migh as well expand the acronym.